src/platform/EFR32/InetPlatformConfig.h

```diff
@@ -49,5 +49,5 @@
 #endif // INET_CONFIG_NUM_TCP_ENDPOINTS
 
 #ifndef INET_CONFIG_NUM_UDP_ENDPOINTS
-#define INET_CONFIG_NUM_UDP_ENDPOINTS 6
+#define INET_CONFIG_NUM_UDP_ENDPOINTS 20
```

Comment:
I am not sure you need this much of them. Did you try with less?

Reply:
Same as we observed in one of our products using Lwip, after we use delayed release of UDP endpoint, we hit running out of endpoint issue, increasing the number of endpoint can mitigate this issue, but it seems a temporary work around solution.

Reply:
This is a SVE cherrypick of an already merged PR. Followups or discussions should be made separately (since already merged, probably a new issue).